app/test_main.py

```diff
@@ -58,5 +58,5 @@ def test_convert_zero(
 
         assert get_human_age(cat_years, dog_years) == result
 
-    def test_type(self) -> None:
+    def test_type_data(self) -> None:
```

Comment:
i mean that you must add test data for this test and test if this data raise exception